app/controllers/user_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UserController < BaseController
+    skip_before_action :authenticate_user , only: [:create]
+
+    def index
+        successfully_borrowed_books = user.books.includes(:borrow_requests).merge(BorrowRequest.successfully_accepted)
```

Comment:
where is user defined? if it is the current_user please rename the function and ensure the function naming is more clear in the future